selfdrive/ui/tests/test_translations.py

```diff
@@ -121,6 +123,34 @@ def test_entities_error(self):
         matches = re.findall(r'@(\w+);', cur_translations)
         self.assertEqual(len(matches), 0, f"The string(s) {matches} were found with '@' instead of '&'")
 
+  def test_bad_language(self):
+    OVERRIDE_WORDS = {'pédale'}
```

Comment:
i'd call this IGNORED_WORDS. also why is this flagged as a bad word?

Reply:
Many cases of incorrectly flagged bad words https://github.com/LDNOOBW/List-of-Dirty-Naughty-Obscene-and-Otherwise-Bad-Words/pull/177